### ISSUE TITLE
Precipitation forecast calculation cannot handle Dutch decimal notation

### ISSUE THREAD

**Issue by corneyl** [mjj4791/python-buienradar#12]
There is a problem with the calculation of the precipitation values because the received JSON from Buienradar can contain decimal values in the Dutch notation, meaning with a `,` as decimal divider. This is an invalid input for the `int()` method.

This is a stacktrace as logged from Home Assistant:
```
Logger: homeassistant
Source: components/buienradar/util.py:147
First occurred: 11:27:13 (2 occurrences)
Last logged: 11:27:13

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/buienradar/util.py", line 147, in async_update
    result = parse_data(
  File "/usr/local/lib/python3.9/site-packages/buienradar/buienradar.py", line 37, in parse_data
    return parse_json_data(content, raincontent,
  File "/usr/local/lib/python3.9/site-packages/buienradar/buienradar_json.py", line 306, in parse_json_data
    data = __parse_precipfc_data(raincontent, timeframe)
  File "/usr/local/lib/python3.9/site-packages/buienradar/buienradar_json.py", line 574, in __parse_precipfc_data
    mmu = 10**(float((int(val) - 109)) / 32)
ValueError: invalid literal for int() with base 10: '104,04313728045622'
```

**Comment by DIY62** [comment]
The cause of this issue seems to be on https://gpsgadget.buienradar.nl/data/raintext? website …

![64455FED-90D5-43F8-9D9E-8584DCF6D203](https://user-images.githubusercontent.com/95056292/143506996-b52359d6-21bc-48f6-bb51-2533e647f210.jpeg)

**Comment by corneyl** [comment]
I forgot to mention it, but I made a PR to fix it: https://github.com/mjj4791/python-buienradar/pull/13

**Comment by mjj4791** [comment]
@corneyl please have a look at your PR #13

**Comment by corneyl** [comment]
Closed by #13
